package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.0",
   "description": "Apache Zeppelin web written in ReactJS",
   "engines": {
-    "node": ">=8",
-    "npm": ">=3"
+    "node": ">=8.9",
+    "npm": ">=6.6"
   },
   "scripts": {
     "preinstall": "node tools/nodeVersionCheck.js",
     "remove-demo": "babel-node tools/removeDemo.js",
     "start-message": "babel-node tools/startMessage.js",
     "prestart": "npm run start-message",
-    "start": "concurrently -k -r -s first \"npm run test:watch\" \"npm run open:src\" \"npm run lint:watch\"",
+    "start": "npm run open:src",
```

Comment:
Can we bring concurrently back a little bit later? Its a bit distracting right now until we stabilize eslint and conventions